ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -170,78 +171,74 @@ Fabric topology is configured under `inventories/eapi-example/group_vars/AVD_FAB
 
 You can also refer to [__Arista Validated Design__ documentation](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/README.md#fabric-topology-variables) to get a description of every single option available.
 
-- Subnet to use for underlay, loopback and vtep:
-
-```yaml
-# Point to Point Network Summary range, assigned as /31 for each
-# uplink interfaces
-# Assign range larger then total [spines * total potential leafs * 2]
-underlay_p2p_network_summary: 172.31.255.0/24
-
-# IP address range for evpn loopback for all switches in fabric,
-# assigned as /32s
-# Assign range larger then total spines + total leafs switches
-overlay_loopback_network_summary: 192.168.255.0/24
-
-# VTEP VXLAN Tunnel source loopback IP for leaf switches, assigned in /32s
-# Assign range larger then total leaf switches
-vtep_loopback_network_summary: 192.168.254.0/24
-```
-
-- MLAG IP information
-
-```yaml
-# mlag pair IP assignment - assign blocks - Assign range larger then
-# total spines + total leafs switches
-mlag_ips:
-  leaf_peer_l3: 10.255.251.0/24
-  mlag_peer: 10.255.252.0/24
-```
-
 Then, you have to describe devices for each role. Don't forget to set management IP here.
 
 - Spine devices
 
 ```yaml
 # Spine Switches
 spine:
-  platform: vEOS-LAB
-  bgp_as: 65001
-  nodes:
-    AVD-SPINE1:
-      id: 1
-      mgmt_ip: 10.73.254.1/24
-    AVD-SPINE2:
-      id: 2
-      mgmt_ip: 10.73.254.2/24
+  defaults:
+    platform: VEOS-LAB
```

Comment:
I saw this to be inconsistent with respect to platform_settings. So I modified it here to make sure everyone uses the appropriate platform in their examples.

Reply:
Looks good.